debian/control

```diff
@@ -18,6 +20,7 @@ Build-Depends:
  libpng-dev,
  libwebp-dev,
  pkg-config,
+ xdg-utils,
```

Comment:
what do we use this package for? I don't see it directly being used but maybe I'm missing something.

Reply:
I'll remove it and see if the test builds work.  I don't remember what it was needed for, but something didn't work in the past without it.

Reply:
xdg-utils used to be required by the old gdk plugin (at least, it was in cmakelists), so the entire plugins folder wouldn't build.  Looks like it isn't needed anymore.